azure-iot-hub-devicesdk/azure/iot/hub/devicesdk/inbox_manager.py

```diff
@@ -48,6 +54,30 @@ def get_c2d_message_inbox(self):
         """
         return self.c2d_message_inbox
 
+    def get_generic_method_call_inbox(self):
+        """Retrieve the Inbox for generic method calls.
+
+        :returns: An Inbox for generic method calls.
+        """
+        return self.generic_method_call_inbox
+
+    def get_named_method_call_inbox(self, method_name):
```

Comment:
>method_name [](start = 42, length = 11)

why not have method_name = None and return either the generic or specific handler here? and have only one get_method_inbox() instead of 2?

Reply:
Good call

---
In reply to: [266185617](https://github.com/Azure/azure-iot-sdk-python-preview/pull/61#discussion_r266185617) [](ancestors = 266185617)